### PR TITLE
NEBULA-2161: Embedded Sandbox adjustments

### PR DESCRIPTION
**Pull request by mayakoneval**
## Context

[JIRA](https://apollographql.atlassian.net/browse/NEBULA-2161?atlOrigin=eyJpIjoiNjRiNTc1ZWIyNTE0NDU3ZWE4ZDZlN2FiOWNlMzI1OTUiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)
[Slack](https://apollograph.slack.com/archives/C025ENJA19Q/p1678288648578999)

## What Changed?

This PR does 3 things
1. passes the packageVersion as query params in the <script tag that requests the embedded Explorer / Sandbox CDN bundle.
2. sends the packageVersion to the embedded Explorer & Sandbox, which are [set up to send these as query params to Studio](https://github.com/apollographql/embeddable-explorer/pull/221). This is redundant data capture just in case.
3. pass `endpointIsEditable: false` to the router, making the landing page embedded Sandbox url not editable

## How to test

erm, can we run the landing pages locally? I made sure tests pass & verified that the template looks as expected

### CODE REVIEW

File: apollo-router/src/services/layers/static_page.rs [hunk 1]
```diff
@@ -135,10 +135,14 @@ impl Default for Sandbox {
 
 #[derive(Template)]
 #[template(path = "sandbox_index.html")]
-struct SandboxTemplate {}
+struct SandboxTemplate<'a> {
+    apollo_router_version: &'a str,
```
Comment (bnjjj) [inline apollo-router/src/services/layers/static_page.rs:139]:
```suggestion
    apollo_router_version: &'static str,
```

File: apollo-router/src/services/layers/static_page.rs [hunk 2]
```diff
@@ -135,10 +135,14 @@ impl Default for Sandbox {
 
 #[derive(Template)]
 #[template(path = "sandbox_index.html")]
-struct SandboxTemplate {}
+struct SandboxTemplate<'a> {
```
Comment (bnjjj) [inline apollo-router/src/services/layers/static_page.rs:138]:
```suggestion
struct SandboxTemplate {
```

**Review by bnjjj — COMMENTED** [review]
Sorry I'm doing the review from my phone so I can't select multiple lines for the suggestion but you can have a static str as it's a constant inserted at the compilation step

**Review by bnjjj — APPROVED** [review]

File: .changesets/fix_sandbox_not_editable_now.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+### Sandbox Explorer endpoint URL no longer editable ([Issue #2729](https://github.com/apollographql/router/pull/2729))
```
Comment (abernix) [inline .changesets/fix_sandbox_not_editable_now.md:1]:
```suggestion
### Sandbox Explorer endpoint URL no longer editable ([PR #2729](https://github.com/apollographql/router/pull/2729))
```

**Pull request merged**